core/src/main/java/com/linecorp/armeria/server/ServiceConfig.java

```diff
@@ -123,6 +125,7 @@ private ServiceConfig(@Nullable VirtualHost virtualHost, Route route,
         this.multipartUploadsLocation = requireNonNull(multipartUploadsLocation, "multipartUploadsLocation");
         this.shutdownSupports = ImmutableList.copyOf(requireNonNull(shutdownSupports, "shutdownSupports"));
         this.defaultHeaders = defaultHeaders;
+        this.serviceErrorHandler = serviceErrorHandler;
```

Comment:
I guess we should never accept `null` here because `ServiceConfigBuilder.build()` already ensures non-nullness.
```suggestion
        this.serviceErrorHandler = requireNonNull(serviceErrorHandler, "serviceErrorHandler");
```